### PR TITLE
cpu-max-usage graph component

### PR DESCRIPTION
**Pull request by FarajiA**
## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-50

## Description:
This is the CPU max usage component with export functionality for Minerva

## Screenshots:
(if applicable)

### CODE REVIEW

File: src/components/cpu/cpu-max-usage/cpu-max-usage.js [hunk 1]
```diff
@@ -119,7 +111,20 @@ export class CpuMaxUsage extends HTMLElement {
         })
         .x(function(data, index){
             return index * 10;
-        })
+            //return new Date(data.time);
```
Comment (kyleondata) [inline src/components/cpu/cpu-max-usage/cpu-max-usage.js]: leftovers?

File: src/components/cpu/cpu-max-usage/cpu-max-usage.js [hunk 2]
```diff
@@ -119,7 +111,20 @@ export class CpuMaxUsage extends HTMLElement {
...
+        });
+    }
+
+    static get observedAttributes() {
```
Comment (kyleondata) [inline src/components/cpu/cpu-max-usage/cpu-max-usage.js]: can we add comments to this function?

File: src/components/cpu/cpu-max-usage/cpu-max-usage.js [hunk 3]
```diff
@@ -119,7 +111,20 @@ export class CpuMaxUsage extends HTMLElement {
...
+        return ['data'];
+    }
+
+    attributeChangedCallback(name, oldValue, newValue) {
```
Comment (kyleondata) [inline src/components/cpu/cpu-max-usage/cpu-max-usage.js]: And to this function?

File: src/components/cpu/cpu-max-usage/cpu-max-usage.js [hunk 4]
```diff
@@ -68,12 +67,22 @@ export class CpuMaxUsage extends HTMLElement {
...
-        chartProps['x'] = d3.scaleLinear().range([0, this.width]);
-        chartProps['y'] = d3.scaleLinear().range([this.height, 0]);
-
-        var xAxis = d3.axisBottom(chartProps['x']);
-        var yAxis = d3.axisLeft(chartProps['y']);
+         // Scale the graph to the dimensions of the svg
+         this.chartProps.x = d3.scaleTime().range([0, this.width]);
+         this.chartProps.y = d3.scaleLinear().range([this.height, 0]);
+        /*
+        this.chartProps.x.domain(d3.extent(this.marshalledData, function(d) {
```
Comment (kyleondata) [inline src/components/cpu/cpu-max-usage/cpu-max-usage.js]: leftovers?

**Comment by FarajiA** [comment]
This branch has gotten out of hand. Will delete and replace

**Pull request closed without merging**